src/componentTest/resources/expectedTemplateContent/service-bus-internal-topic-datasync/src/main/resources/logback.xml

```diff
@@ -1,12 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <include resource="com/blackbaud/boot/logback/common-vsts.xml" />
+    <include resource="com/blackbaud/boot/logback/common-logback.xml" />
+    <include resource="com/blackbaud/async/logback/common-async.xml"/>
```

Comment:
should this be common-servicebus ?

Reply:
no, there is a shared file for servicebus and eventhubs; i talked to shafath about this, my initial reaction was to have two separate files but each would define `<logger name="com.microsoft.azure" level="WARN"/>` which might be confusing, so it seems like a single file but pulling in individual rule sets (one for service-bus, one for event-hubs) was the way to go

Reply:
maybe should rename from `common-async` to `common-async-vsts`?  though i doubt we'll be adding common-kafka here so 🤷‍♂️